### DISCUSSION TITLE
使用有道翻译引擎每次重新打开  webstorm 都要重新填写密钥

### DISCUSSION THREAD

**Discussion by danyun123** [YiiGuxing/TranslationPlugin#5616] [Q&A]
webstorm 版本为：2023.3.6 ；之前使用的时候填写一次后就永久有效了，更新了一次该插件后就需要每次重新填写了。

**Comment by YiiGuxing** [comment]
按照这个方式试试：https://intellij-translation.yiiguxing.top/#/faq?id=faq-save-key

**Reply by danyun123** [reply]
解决了😁

**Reply by GavinZ1314** [reply]
这个打不开了，能再给个链接么，我这是使用百度翻译，每次打开idea都要填密钥

**Reply by YiiGuxing** [reply]
https://intellij-translation.yiiguxing.top/#/faq?id=faq-save-key

**Reply by GavinZ1314** [reply]
> https://intellij-translation.yiiguxing.top/#/faq?id=faq-save-key

谢谢
